Meetings/2021-12-08-meeting-minutes.md

```diff
@@ -0,0 +1,49 @@
+# Maintainers' Meeting 2021-21-08
+
+## Meeting Attendees
+
+- Rob Prouse - @rprouse
+- Mattias Karlsson - @devlead
+- Calvin A. Allen - @CalvinAllen
+- Dan Siegel - @dansiegel
+- James Newton-King - @JamesNK
+- James Turner - @Turnerj
+- Pascal Berger - @pascalberger
+- Nils Andresen - @nils-a
+
+## Recording
+
+**TBD**
```

Comment:
@rprouse Not sure if this is a "good" place to discuss, but:
The recording for the last (second to last? Well, the "other" meeting..) are also still "TBD". Is that a "need the time to upload" issue, or rather a "don't know where to put it" issue? 
Is there anything someone can do to help?

Reply:
It is a find a don't know where to store it issue. I tried sharing from the foundation OneDrive but external sharing is blocked. I want to get that changed. We also discussed YouTube, but I'm not sure meeting recordings should be in the main Foundation YouTube feed.

Reply:
Could do an unlisted video and just link it here? Wouldn't appear in anyone's subscription feeds then.

Reply:
I'll ask about that, thanks. I'm not going to hold up merging this though, so I'm going to resolve.